packages/keyring-eth-qr/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@metamask/eth-qr-keyring",
+  "version": "1.0.0",
```

Comment:
Should this start off at 1.0.0? I'm not sure how releases work in this monorepo but if it's the same as `core` then we would set the initial version in a release PR.

``` suggestion
  "version": "0.0.0",

Reply:
Definitely, done in 5ab4c618aa82